apps/myaccount/src/store/actions/authenticate.ts

```diff
@@ -301,6 +306,43 @@ export const initializeAuthentication = () =>(dispatch)=> {
     };
 
     const initialize = (response?: any): void => {
+
+        // Define endpoints
+        let serverOrigin = window["AppUtils"].getConfig()?.serverOrigin;
+        let authorizeURL = window["AppUtils"].getConfig().idpConfigs?.authorizeEndpointURL;
+        let oidcSessionIFrameURL = window["AppUtils"].getConfig().idpConfigs?.oidcSessionIFrameEndpointURL;
+        let tokenURL = window["AppUtils"].getConfig().idpConfigs?.tokenEndpointURL;
+        let tokenRevokeURL = window["AppUtils"].getConfig().idpConfigs?.tokenRevocationEndpointURL;
+        let jwksURL = window["AppUtils"].getConfig().idpConfigs?.jwksEndpointURL;
+        let wellKnownEndpoint = window["AppUtils"].getConfig().idpConfigs?.wellKnownEndpointURL;
+        let logoutURL = window["AppUtils"].getConfig().idpConfigs?.logoutEndpointURL;
+
+        /* Check whether the host is different compared to the server origin. If different, we assume that the app
+        is accessed through a custom domain. */
+        const serverHost = serverOrigin.split("//")[1];
+        if (serverHost !== window.location.host) {
```

Comment:
What happens if the app is hosted on a separate environment, maybe on a different tomcat server?

Reply:
This wouldn't work in that case. I tried to use client Origin instead but that always gets changed to the domain that we access.

Reply:
As per the discussion had with @brionmario and @jeradrutnam, even if the app is hosted on a separate environment, it'll be mapped to the asgardio domain, hense the logic will still be valid.